app/helpers/proxmox_vm_cloudinit_helper.rb

```diff
@@ -41,4 +41,49 @@ def parse_server_cloudinit(args)
     end
     cloudinit_h
   end
+
+  def create_cloudinit_iso(vm_name, configs)
+    iso = File.join('/var/lib/vz/template/iso', "#{vm_name}_cloudinit.iso")
+    arguments = ["-output #{iso}", '-volid', 'cidata', '-joliet', '-rock']
+    ssh = Fog::SSH.new(URI.parse(fog_credentials[:proxmox_url]).host, fog_credentials[:proxmox_username].split('@')[0], { password: fog_credentials[:proxmox_password] })
+    wd = ssh.run([%({mktemp -d})]).first.stdout.chomp
```

Comment:
what happens if the ssh connection can not be established?

Reply:
It will throw error from fog side.

Reply:
then we need to catch this error and don't proceed with other stuff?

Reply:
That's also done on fog side. It raises the error and we don't proceed further.